apis/generate.mk

```diff
@@ -1,9 +1,10 @@
 clean_targets += .clean-apis
 
 .PHONY: generate-xrd
-generate-xrd: ## Generates the XRDs using Kubebuilder
+generate-crd: ## Generates the XRDs using Kubebuilder
```

Comment:
Aren't we generating composites here? If that's the case I would leave XRD instead of CRD. CRD is a broader term.

Reply:
Yeah naming is hard.  Technically the command now generated CRDs that are then turned into XRDs in Jsonnet. So I renamed the command. But I guess this confuses more than it helps..

Reply:
Then I would leave `generate-crd` and add more details in the comment.